homework/05LessonFive.js

```diff
@@ -63,14 +63,16 @@ for (forLoopCounter = 0; forLoopCounter > -10; forLoopCounter--) {
 
 // Problem 8:
 // Fill in forLoopArrayResult so that it matches forLoopArray
-var forLoopArrayResult = [];
+var forLoopArrayResult = [0,-1,-2,-3,-4,-5,-6,-7,-8,-9];
 
 // ****
 // Concept Checkpoint
 //
 // Write your answer in comments
 //
 // What are loops? Why do we use them?
+// The loops are ciclos to repeate something many times while a condition is true.
```

Comment:
Not bad! It's also important to note that it reduces the amount of code you have to write. You can write three or four lines and have it run as many times as you like!